cmd/cli/get/get_test.go

```diff
@@ -125,6 +126,7 @@ func (s *GetSuite) getDockerRunArgs(extraArgs []string) []string {
 // it makes it's own folder to put the results in and does not splat results
 // all over the current directory
 func (s *GetSuite) TestDockerRunWriteToJobFolderAutoDownload() {
+	s.T().Skip("--download not supported in v2")
```

Comment:
why is `--download` not supported?

Reply:
The printer logic used by V2 CLIs does no have support for downloads - e.g. the job run command doesn't support these auto download flags. We can either address this in a follow on where we unify auto download flags across all CLIs. Or I can attempt to address this now and preserve the existing flag. Which would you prefer?

Reply:
I see. My preference to preserve the existing flag and address this as part of this change. I don't want merging this PR to block the release pipeline, and we shouldn't release without it

Reply:
Tracking for this here. Will implement once the wasm CLI is migrated: https://github.com/bacalhau-project/bacalhau/issues/4058